src/TestingUtils/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
Suggest moving everything from Microsoft.AspNetCore.Testing up into this folder. Might be good to rename TestingUtils/ -> Testing/ as well.